simulation/halsim_ws_core/src/main/native/cpp/WSProvider_DriverStation.cpp

```diff
@@ -126,7 +137,16 @@ void HALSimWSProviderDriverStation::OnNetValueChanged(const wpi::json& json) {
     auto joysticks = it.value();
     for (auto jit = joysticks.cbegin(); jit != joysticks.cend(); ++jit) {
       auto stick = jit.value();
-      auto num = std::stoi(jit.key());
+
+      int num;
```

Comment:
I think you should init here since if it throws the value of num is unknown.

Reply:
shouldn't need to since the `catch` does a `continue` on the loop

Reply:
Agreed. I missed the continue but still should have num initialized in case someone modifies the code in the future.